source/delegate/package.json

```diff
@@ -29,11 +29,12 @@
   "devDependencies": {
     "@airswap/order-utils": "0.3.15",
     "@airswap/test-utils": "0.1.3",
+    "@airswap/transfers": "0.1.0",
     "solidity-coverage": "^0.6.3"
   },
   "dependencies": {
     "@airswap/indexer": "2.6.7",
     "@airswap/swap": "3.3.5",
-    "openzeppelin-solidity": "2.3"
+    "openzeppelin-solidity": "2.4"
```

Comment:
is this upgrade because you want some of the new functionality? Or just because an upgrade was available?

Reply:
because i wanted to functionality specificially SafERC20.sol and better revert messages about lack of balances and approvals.